src/selectors/events.js

```diff
@@ -144,32 +124,12 @@ export const groupPerformancesByPeriod = (
   performances: Performance[]
 ): PerformancePeriods => {
   if (performances == null) return [];
-  const sections = performances
-    .sort(
-      (a: Performance, b: Performance) =>
-        parseDate(a.fields.startTime[locale]) -
-        parseDate(b.fields.startTime[locale])
-    )
-    .reduce(
-      ({ periods, buffer }, performance) => {
-        if (buffer.length < 1) return { periods, buffer: [performance] };
-
-        const previous: Performance = buffer[buffer.length - 1];
-
-        if (
-          getTimePeriod(parseDate(previous.fields.startTime[locale])) !==
-          getTimePeriod(parseDate(performance.fields.startTime[locale]))
-        )
-          return { periods: [...periods, buffer], buffer: [performance] };
-
-        return { periods, buffer: [...buffer, performance] };
-      },
-      { periods: [], buffer: [] }
-    );
-
-  return sections.buffer.length > 0
-    ? [...sections.periods, sections.buffer]
-    : sections.periods;
```

Comment:
Refactoring these two large functions might seem unrelated, but it allowed me to remove the need for `parseDate`, and `differenceInCalendarDays` functions. These were perfectly possible in `luxon`, but I saw an opportunity to simplify.